packages/core/src/modules/oob/repository/OutOfBandRecord.ts

```diff
@@ -56,17 +60,22 @@ export class OutOfBandRecord extends BaseRecord<DefaultOutOfBandRecordTags> {
       this.reusable = props.reusable ?? false
       this.mediatorId = props.mediatorId
       this.reuseConnectionId = props.reuseConnectionId
-      this._tags = props.tags ?? {}
+      this._tags = props.tags ?? {
+        recipientKeyFingerprints: props.outOfBandInvitation
+          .getInlineServices()
+          .map((s) => s.recipientKeys)
+          .reduce((acc, curr) => [...acc, ...curr], [])
+          .map((didKey) => DidKey.fromDid(didKey).key.fingerprint),
+      }
     }
   }
 
-  public getTags() {
+  public getTags(): Tags<DefaultOutOfBandRecordTags, CustomOutOfBandRecordTags> {
```

Comment:
This should be inferred I think. Is it giving issues?
```suggestion
  public getTags() {
```

Reply:
explicit types removed